queue/util.lua

```diff
@@ -0,0 +1,41 @@
+local fiber = require('fiber')
+
+-- MAX_TIMEOUT == 100 years
+local MAX_TIMEOUT      = 365 * 86400 * 100
+-- Set to TIMEOUT_INFINITY
+-- instead returns time for next event
+local TIMEOUT_INFINITY = 18446744073709551615ULL
+
+local function time(tm)
```

Comment:
It transforms what to what? The same question for event_time().

Since you're in the context now, I would ask you to add a comment for those functions.

Sorry for nitpicking.

Reply:
Accepted.